src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -59,10 +59,16 @@
         = "org.jitsi.videobridge.DISABLE_NACK_TERMINATION";
 
     /**
+     * Configuration property for number of streams to cache
+     */
+    final static String ENABLE_LIPSYNC_HACK_PNAME
+        = VideoChannel.class.getName() + ".ENABLE_LIPSYNC_HACK";
+
+    /**
      * The name of the property which controls whether {@link VideoChannel}s
      * periodically log statistics related to oversending data.
      */
-    public static final String LOG_OVERSENDING_STATS_PNAME
+    private static final String LOG_OVERSENDING_STATS_PNAME
```

Comment:
Maybe I'm missing something, but it seems like it's not used elsewhere. So why have it public?